schema-engine/sql-migration-tests/tests/apply_migrations/mod.rs

```diff
@@ -354,7 +354,7 @@ fn migrations_should_fail_when_the_script_is_invalid(api: TestApi) {
                 t if t.contains(Tags::Mysql) => "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near \'^.^)_n\' at line 1",
                 t if t.contains(Tags::Mssql) => "Incorrect syntax near \'^\'.",
                 t if t.contains(Tags::Postgres) => "ERROR: syntax error at or near \"^\"",
-                t if t.contains(Tags::Sqlite) => "unrecognized token: \"^\"",
+                t if t.contains(Tags::Sqlite) => "unrecognized token: \"^\" in \n\nSELECT (^.^)_n;\n at offset 10",
```

Comment:
What is causing this change?

Reply:
Underlying SQLite change! nothing to do with rusqlite itself afaik